src/activitylog/activitylog.css

```diff
@@ -1,36 +1,179 @@
+@font-face {
+  font-family: source-sans;
+  src: url(/fonts/SourceSansPro-Regular.ttf);
+}
+
 *,
 *:before,
 *:after {
   padding: 0;
   margin: 0;
   box-sizing: border-box;
+  font-family: source-sans;
+}
+
+.failure {
+  background-color: #961616;
+  padding: 10px;
+  color: #ffffff;
+}
+
+.filter-wrapper filter-option:not(:last-child) {
+  margin: 0 5px;
+}
+
+.filter-wrapper filter-keyword {
+  margin-right: 5px;
+}
+
+.filter-wrapper filter-timestamp {
+  margin-left: 5px;
+}
+
+.menu-wrapper {
+  padding: 5px 10px;
+}
+input[type='file'] {
+  display: none;
+}
+
+.content-wrapper {
+  display: grid;
+}
+
+.separator {
+  border-right: 2px solid #dfdfdf;
+  height: 100%;
+  max-height: 40px;
+}
+
+.menu-wrapper {
+  display: grid;
+  grid-template-columns: 25px 2px auto 2px 25px;
+  grid-column-gap: 10px;
+  align-items: center;
+}
+
+.options-dropdown {
+  background: #fff;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.25), 0 6px 20px 0 rgba(0, 0, 0, 0.25);
+  position: absolute;
+  right: 8px;
+  width: 150px;
+  z-index: 10001;
+  margin-top: 15px;
+}
+
+.menu-wrapper.load-logs {
+  grid-template-columns: auto;
+}
+
+.options-dropdown > *:hover {
+  background-color: #a2eee3;
+}
+
+.clear-logs-btn,
+.options-btn,
+.save-log-btn,
+.load-file {
+  cursor: pointer;
+}
+
+.options-btn,
+.clear-logs-btn {
+  height: 24px;
+  width: 24px;
+}
+
+.options-btn {
+  background: url('/icons/settings.svg');
 }
 
-.activity-log-wrapper {
-  position: relative;
+.clear-logs-btn {
+  background: url('/icons/clear.svg');
 }
 
-.container {
+.save-log-btn {
+  background: url('/icons/save.svg') no-repeat 5px center / auto;
+}
+
+.load-file-btn {
+  background: url('/icons/upload.svg') no-repeat 5px center / auto;
+}
+.options-dropdown > * {
+  padding-left: 35px;
   width: 100%;
-  padding: 0 10px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
 }
 
-#saveLogBtn {
-  margin: 20px 0;
+.options-dropdown img {
+  width: 24px;
+  height: auto;
+  padding-right: 8px;
 }
 
-.failure {
-  background-color: #961616;
-  padding: 10px;
+.options-dropdown:after,
```

Comment:
Use `::after` and `::before` instead of `:after` and `:before`.

They are equivalent and will be supported because of back-compat, but a singular `:` is now used for pseudo-classes, whereas a double `::` is used for pseudo-elements.

For more context, see https://stackoverflow.com/questions/7327326/what-is-the-difference-between-before-and-before

Reply:
Yes used `::` instead of single `:`